src/openrct2/ReplayManager.cpp

```
@@ -752,42 +861,129 @@ namespace OpenRCT2
+                    }
+                    serialiser << ReplayEntryType::End;
+                }
+                else
                 {
-                    SerialiseCommand(serialiser, const_cast<ReplayCommand&>(command));
+                    ReplayEntryType type;
+                    try
+                    {
+                        while (true)
+                        {
+                            serialiser << type;
```

Cppcheck (2.19.0)

[error] Uninitialized variable

(uninitvar)